### PR TITLE
Talks - slides added to documentation.

### PR DESCRIPTION
**Pull request by timbl**
Work in progress
Plan is for talks on forms at least.

### CODE REVIEW

File: documentation/talks/forms/FormsTalk.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+<br /><em>Hit the space bar or swipe left for next slide</em></p>
+</div>
+
+<div class="slide">
+<h1>Declarative User Interface: Forms and Apps in RDF</h1>
+
+<ul>
+<li>Declaratve design is good.</li>
+<li>Compose apps and parts of apps out of <b>components</b>/li>
+<li>High-level design allows <b>multiple</b> compaitible implementations</li>
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:67]:
```suggestion
<li>High-level design allows <b>multiple</b> compatible implementations</li>
```

File: documentation/talks/forms/FormsTalk.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+<li>Why not use that for User Interface?</li>
+</ul>
+</div>
+
+<div class="slide">
+<h1>What would that look like?</h1>
+
+<pre>
+  @@ add a really small form with  just a set of questions - name steing, children number, enrolled date, interested boolean.
+  Forms part of the talkshould be on the web, maybe in the talk itself/examples or in other places where we have persistent curated set of examples.
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:78]:
```suggestion
  @@ add a really small form with just a set of questions - name string, children number, enrolled date, interested boolean.
  Forms part of the talk should be on the web, maybe examples in the talk itself, or in other places where we have persistent curated set of examples.
```

File: documentation/talks/forms/FormsTalk.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+</pre>
+</div>
+
+
+<div class="slide">
+<h1>Components are form Fields or structures</h1>
+<h3>Fields<h3>
+<ul>
+  <li>@@ add fileds
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:87]:
```suggestion
  <li>@@ add fields
```

File: documentation/talks/forms/FormsTalk.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  </li>
+</ul>
+
+</div>
+
+<div class="slide">
+<h1>Single Line Text field</h1>
+<p>Form fields are set up to write linked data into Solid pods.</p>
+<p>Each one adds (typically) a new edge to the data graph at run time</p>
+<p>So e give the predicate which will be used in the instance of the field.</p>
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:102]:
```suggestion
<p>Each one adds (typically) a new edge to the data graph at run time.</p>
<p>So we give the predicate which will be used in the instance of the field.</p>
```

File: documentation/talks/forms/FormsTalk.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+<div class="slide">
+<h1>Structures</h1>
+<p>Form componetnts which are structures combine
+  several fields</p>
+</div>
+
+<div class="slide">
+<h1>Group</h1>
+<p>A group is just set of components in order</p>
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:119]:
```suggestion
<p>A group is just a set of components in an order</p>
```

File: documentation/talks/forms/FormsTalk.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+@@ example, of form data and UI
+</div>
+
+<div class="slide">
+<h1>@@ Add examples for other structures</h1>
+</div>
+
+<div class="slide">
+<h1>Example: A contact card form</h1>
+@@ on the left the form data, on the right the form rendred
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:129]:
```suggestion
@@ on the left the form data, on the right the form rendered
```

File: documentation/talks/forms/FormsTalk.html [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+<h1>Going meta -- designing forms using the form of forms 2</h1>
+@@ here we are editing the really small form we started with.
+</div>
+
+<div class="slide">
+<h1>Questions?</h1>
+</div>
+<div class="slide">
+<h1>@@ The Autocomplete field</h1>
+<p>This form field allows users to select things from lists in public databases, things like Languages, Organizations, Occupations, and Skills</p>
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:153]:
```suggestion
<p>This form field allows users to select things &mdash; like Languages, Organizations, Occupations, and Skills &mdash; from lists in public databases,</p>
```

File: documentation/talks/forms/FormsTalk.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+<p>This is not covered in this talk, but a later one.</p>
+<p>(Ther is a @@ref blog about is which I could run though if interested)</p>
```
Comment (TallTed) [inline documentation/talks/forms/FormsTalk.html:155]:
```suggestion
<p>(There is a @@ref blog about it which I could run though if interested)</p>
```

File: documentation/talks/forms/slidey.html [hunk 9]
```diff
@@ -0,0 +1,916 @@
...
+<div class="slide">
+<h1>Slide Shows in HTML and XHTML</h1>
+
+<ul>
+<li>You can now create accessible slide shows with ease</li>
+
+<li>Works across browsers and is operated like PowerPoint
+
+<ul>
+<li>Advance to next slide with mouse click, space bar or swipe left</li>
```
Comment (TallTed) [inline documentation/talks/forms/slidey.html:65]:
```suggestion
<li>Advance to next slide with mouse click, space bar, or swipe left</li>
```

File: documentation/talks/forms/slidey.html [hunk 10]
```diff
@@ -0,0 +1,916 @@
...
+
+<li>Move forward/backward between slides with Cursor Left,
+Cursor Right, <strong>Pg Up</strong> and <strong>Pg Dn</strong>
+keys, or swipe left or right</li>
```
Comment (TallTed) [inline documentation/talks/forms/slidey.html:69]:
```suggestion
Cursor Right, <strong>Pg Up</strong>, or <strong>Pg Dn</strong>
keys; or swipe left or right</li>
```

File: documentation/talks/forms/slidey.html [hunk 11]
```diff
@@ -0,0 +1,916 @@
...
+
+<li><strong>Home</strong> key for first slide, <strong>End</strong>
+ key for last slide</li>
+
+<li>The "<strong>C</strong>" key for an automatically generated
+table of contents, or click on "contents" on the toolbar or
```
Comment (TallTed) [inline documentation/talks/forms/slidey.html:75]:
```suggestion
table of contents, or click on "contents" on the toolbar, or
```

File: documentation/talks/forms/slidey.html [hunk 12]
```diff
@@ -0,0 +1,916 @@
...
+
+<li>Function <strong>F11</strong> to go full screen and back</li>
+
+<li>The "<strong>F</strong>" key toggles the display of the footer</li>
+
+<li>The "<strong>A</strong>" key toggles display of current vs all
+slides
+
+<ul>
+<li>Try it now to see how to include notes for handouts (this is
```
Comment (TallTed) [inline documentation/talks/forms/slidey.html:86]:
```suggestion
<li>Try it now, to see how to include notes for handouts (this is
```

File: documentation/talks/forms/slidey.html [hunk 13]
```diff
@@ -0,0 +1,916 @@
...
+(<a href="http://www.csail.mit.edu/"><acronym title=
+"Massachusetts Institute of Technology">MIT</acronym></a>, <a href=
+"http://www.ercim.org/"><acronym title=
+"European Research Consortium for Informatics and Mathematics">ERCIM</acronym></a>,
+<a href="http://www.keio.ac.jp/">Keio</a>), All Rights
+Reserved.</p>
+</div>
+
+<div class="handout">
+<p>For handouts, its often useful to include extra notes using a
```
Comment (TallTed) [inline documentation/talks/forms/slidey.html:122]:
```suggestion
<p>For handouts, it's often useful to include extra notes using a
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Fixed some typos and similar. There will be more, beyond where I got to.

**Comment by timbl** [comment]
I have abandoned this branch at the more logical place to put the talk is in solid-II

**Pull request closed without merging**